molecule/centos/molecule.yml

```diff
@@ -14,6 +14,7 @@ platforms:
     privileged: true
     override_command: false
     dns_servers:
+      - 129.206.100.126
```

Comment:
Adding the default Uni Heidelberg DNS Server testing means that it is called everytime in our test. Is that fine?